From 4efa710b230c93d54f27f9f33900629a8a8dcde6 Mon Sep 17 00:00:00 2001
From: Robert Lehmann <robert.lehmann@sitec-systems.de>
Date: Thu, 7 Jan 2016 17:06:31 +0100
Subject: [PATCH 09/11] S Core: imximage: Add boot options

Add the possibility to boot from spinor or from sd.

Ticket #871
---
 board/freescale/mx6sxscore/imximage.cfg | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/board/freescale/mx6sxscore/imximage.cfg b/board/freescale/mx6sxscore/imximage.cfg
index 312203f..19d0a00 100644
--- a/board/freescale/mx6sxscore/imximage.cfg
+++ b/board/freescale/mx6sxscore/imximage.cfg
@@ -23,7 +23,11 @@ IMAGE_VERSION 2
  * spi/sd/nand/onenand, qspi/nor
  */
 
+#ifdef CONFIG_SYS_BOOT_SPINOR
 BOOT_FROM	spi
+#else
+BOOT_FROM   sd
+#endif
 
 #ifdef CONFIG_USE_PLUGIN
 /*PLUGIN    plugin-binary-file    IRAM_FREE_START_ADDR*/
-- 
1.9.1

